domain/src/main/java/com/example/pinterbest/domain/usecases/GetCheckAuthUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.pinterbest.domain.usecases
+
+import com.example.pinterbest.domain.repositories.AuthRepository
+import javax.inject.Inject
+
+class GetCheckAuthUseCase @Inject constructor(
+    private val authRepository: AuthRepository
+) {
+    suspend operator fun invoke() = authRepository.getCheckAuth()
```

Comment:
nit: marking the fun as `operator` it's not necessarily

Reply:
Convenient use of use-cases and readability of the code. Let's leave it like this for now